lib/directory.nix

```diff
@@ -18,7 +18,7 @@ in
         # consider that it comes from a folder without read access only as in
         # Nix
         mkdir -p "$DIRECTORY"/staging
-        cp ${jupyter}/lib/python3.7/site-packages/jupyterlab/staging/yarn.lock "$DIRECTORY"/staging
+        cp ${jupyter}/lib/${pkgs.python3.libPrefix}/site-packages/jupyterlab/staging/yarn.lock "$DIRECTORY"/staging
```

Comment:
Consider using `${pkgs.python3.sitePackages}/jupyterlab` instead of `${pkgs.python3.libPrefix}/site-packages/jupyterlab`.

Reply:
Thanks